pages/course/[courseId]/unit/HTML/1.tsx

```diff
@@ -0,0 +1,95 @@
+import Card from "../../../../../components/coding/Card";
+import Sidebar from "../../../../../components/coding/studentPortal/Sidebar";
+import ProgressBar from "../../../../../components/coding/studentPortal/ProgressBar";
+import Navbar from "../../../../../components/ui/Navbar";
+import { ResourceRow } from "../../../../../components/coding/studentPortal/ResourceRow";
+import { Button } from "../../../../../components/ui/Button";
+import Layout from "../../../../../components/coding/studentPortal/Layout";
+
+const HTML1 = () => {
```

Comment:
we shouldn't be linking to this page anymore. I moved it to studentPortal/intro/HTML/1. If you merge this in we're going to have a copy of this pages.